### ISSUE TITLE
Experiment details in paper

### ISSUE THREAD

**Issue by Kaihua1203** [jwmao1/MedSegFactory#11]
Hi, Thank you for excellent work  and great contribution in community!

Except for the requirement of releasing test dataset for generation result evaluation, I also have the problem about experiment. 
1. How do you train ControlNet for **image-mask pair** generation? 
2. Did you train separate ControlNet for image and mask generation, or was this handled jointly?
3. Regarding Stable Diffusion 1.5 fine-tuning: did you use LoRA, and if so, were different LoRAs trained on different datasets or a unified one?

**Comment by jwmao1** [comment]
Thanks for your interest in our work! Actually, we do not train ControlNet or LoRA in MedSegFactory. Instead, we design JCA within Stable Diffusion to synchronously generate medical images and their corresponding masks. Please refer to the paper for more technical details.
